test/e2e/util_test.go

```diff
@@ -872,144 +871,33 @@ func deploymentReplicas(replicas int32) predicateFunc {
-//
-// plan := &InstallPlan{}
-// plan.SetNamespace("ns")
-// plan.SetName("install-123def")
-// plan.GetObjectKind().SetGroupVersionKind(schema.GroupVersionKind{
-//		Group:   GroupName,
-//		Version: GroupVersion,
-//		Kind:    InstallPlanKind,
-// })
-// Eventually(Apply(plan, func(p *v1alpha1.InstallPlan) error {
-//		p.Spec.Approved = true
-//		return nil
-// })).Should(Succeed())
-func Apply(obj Object, changeFunc interface{}) func() error {
-	// Ensure given object is a pointer
-	objType := reflect.TypeOf(obj)
-	if objType.Kind() != reflect.Ptr {
-		panic(fmt.Sprintf("argument object must be a pointer"))
-	}
-
-	// Ensure given function matches expected signature
-	var (
-		change     = reflect.ValueOf(changeFunc)
-		changeType = change.Type()
-	)
-	if n := changeType.NumIn(); n != 1 {
-		panic(fmt.Sprintf("unexpected number of formal parameters in change function signature: expected 1, present %d", n))
-	}
-	if pt := changeType.In(0); pt.Kind() != reflect.Interface {
-		if objType != pt {
-			panic(fmt.Sprintf("argument object type does not match the change function parameter type: argument %s, parameter: %s", objType, pt))
-		}
-	} else if !objType.Implements(pt) {
-		panic(fmt.Sprintf("argument object type does not implement the change function parameter type: argument %s, parameter: %s", objType, pt))
-	}
-	if n := changeType.NumOut(); n != 1 {
-		panic(fmt.Sprintf("unexpected number of return values in change function signature: expected 1, present %d", n))
-	}
-	var err error
-	if rt := changeType.Out(0); !rt.Implements(reflect.TypeOf((*error)(nil)).Elem()) {
-		panic(fmt.Sprintf("unexpected return type in change function signature: expected %t, present %s", err, rt))
-	}
-
-	// Determine if we need to apply a status subresource
-	// FIXME: use discovery client instead of relying on storage struct fields.
-	_, applyStatus := objType.Elem().FieldByName("Status")
-
-	var (
-		bg     = context.Background()
-		client = ctx.Ctx().Client()
-	)
-	key, err := controllerclient.ObjectKeyFromObject(obj)
-	if err != nil {
-		panic(fmt.Sprintf("unable to extract key from resource: %s", err))
-	}
-
-	// Ensure the GVK is set before patching
-	SetDefaultGroupVersionKind(obj)
-
-	return func() error {
-		changed := func(obj Object) (Object, error) {
-			cp := obj.DeepCopyObject().(Object)
-			if err := client.Get(bg, key, cp); err != nil {
-				return nil, err
-			}
-			// Reset the GVK after the client call strips it
-			SetDefaultGroupVersionKind(cp)
-			cp.SetManagedFields(nil)
-
-			out := change.Call([]reflect.Value{reflect.ValueOf(cp)})
-			if len(out) != 1 {
-				panic(fmt.Sprintf("unexpected number of return values from apply mutation func: expected 1, returned %d", len(out)))
-			}
-
-			if err := out[0].Interface(); err != nil {
-				return nil, err.(error)
-			}
+const (
+	cvoNamespace      = "openshift-cluster-version"
+	cvoDeploymentName = "cluster-version-operator"
+)
 
-			return cp, nil
-		}
+func toggleCVO() {
```

Comment:
this function should have been removed when the `Operators` API was released under the `v1` group.